Force the python executable version to run sphinx_build, so it use the right
one, with the right pythonpath.

--- CMakeLists.txt.orig	2013-04-02 14:40:29.000000000 +0200
+++ CMakeLists.txt	2013-07-04 15:18:33.174014000 +0200
@@ -161,7 +161,7 @@
 	SET (SPHINX_LATEX_OUTPUT ${CMAKE_DOC_OUT}/tex)
 	SET (SPHINX_MAN_OUTPUT ${CMAKE_DOC_OUT}/man)
 	SET (SPHINX_FALSE_PYTHON_ENV ${CMAKE_CURRENT_BINARY_DIR}/fakeenv)
-	SET (PYTHON_DOC_ENV env PYTHONPATH=${CMAKE_CURRENT_BINARY_DIR}:${CMAKE_CURRENT_BINARY_DIR}/src:${SPHINX_FALSE_PYTHON_ENV}:${CMAKE_CURRENT_SOURCE_DIR}/src:${CMAKE_CURRENT_SOURCE_DIR}/testing:${CMAKE_CURRENT_SOURCE_DIR}/bindings/pymorse/src/:$$PYTHONPATH PYTHONDONTWRITEBYTECODE="morse" LANG="C.UTF-8" LC_ALL="C.UTF-8")
+	SET (PYTHON_DOC_ENV env PYTHONPATH=${CMAKE_CURRENT_BINARY_DIR}:${CMAKE_CURRENT_BINARY_DIR}/src:${SPHINX_FALSE_PYTHON_ENV}:${CMAKE_CURRENT_SOURCE_DIR}/src:${CMAKE_CURRENT_SOURCE_DIR}/testing:${CMAKE_CURRENT_SOURCE_DIR}/bindings/pymorse/src/:$$PYTHONPATH PYTHONDONTWRITEBYTECODE="morse" LANG="C.UTF-8" LC_ALL="C.UTF-8" ${PYTHON_EXECUTABLE})
 	SET (SPHINX_BUILD_ENV ${PYTHON_DOC_ENV} ${SPHINX_BUILD})
 
 	# Copy the documentation in builddir, so we can include the generated part too
@@ -174,7 +174,7 @@
 	# Generate the documentation from the different components
 	ADD_CUSTOM_TARGET(
 		generate_doc ALL
-		COMMAND ${PYTHON_DOC_ENV} ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/doc/tools/generate_doc
+		COMMAND ${PYTHON_DOC_ENV} ${CMAKE_CURRENT_SOURCE_DIR}/doc/tools/generate_doc
 		WORKING_DIRECTORY ${SPHINX_SRC}/morse/user/
 	)
 
